docs/runbooks/VMOutdatedMachineType.md

```diff
@@ -0,0 +1,41 @@
+# VMOutdatedMachineType
+
+## Meaning
+This alert fires when there are virtual machines (VMs) using machine types
+that are no longer supported.
```

Comment:
we usually use rhel machine types, and rhel drops support every 2 major version gap.
so if in 4.19 we would use rhel10 images, they will contain breaking changes that wont be compatible anymore with old machines using rhel8.
I feel like it will be a little bit controversial to write rhel stuff in upstream
@sradco WDYT?

Reply:
Correct. No need to add here the version numbers.